src/main/java/de/srendi/advancedperipherals/common/configuration/WorldConfig.java

```diff
@@ -20,6 +21,7 @@ public WorldConfig() {
         enableVillagerStructures = builder.comment("Enable the villager structures for the computer scientist.").define("enableVillagerStructures", true);
         givePlayerBookOnJoin = builder.comment("Gives the ap documentation to new players.").define("givePlayerBookOnJoin", true);
         villagerStructureWeight = builder.comment("The weight of the villager structures.").defineInRange("villagerStructureWeight", 10, 0, 16000);
+        enableWanderingTraderTrades = builder.comment("Enable new wandering trader trades.").define("enableWanderingVillagerTrades", true);
```

Comment:
Shouldn't it be `enableWanderingTraderTrades` and not `...Villager...`

Reply:
Quite right